.gitignore

```diff
@@ -119,3 +119,6 @@ dist
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+
+# Final static build files
+scholly/base/build
```

Comment:
out of curiosity, why are we ignoring this file?

Reply:
I believe this is the generated `build` after `npm run build` is executed. I think we don't want this in the git but I thought we ignored this already. Thanks for adding it